src/index.html

```diff
@@ -6,7 +6,561 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Фото
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Видео
+            </a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="Моноблок"
+            class="card__img"
+          >
+
+          <div class="card__description">
+            <span class="card__product-name">
+              Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+            </span>
+
+            <span class="card__product-code">
+              Код товара: 195434
+            </span>
+
+            <div class="card__reviews">
+
+              <ul class="stars--4 stars">
+                <li class="stars__star">
+                </li>
```

Comment:
```suggestion
                <li class="stars__star"></li>
```

в таком случае лучше писать на 1 строке